Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest AS build
+FROM alpine:3 as node-builder
 
 COPY . /src
```

Comment:
btw, we might link sources rather than cp (I think this will slightly reduce runtime overhead for build; I know this is less relevant otherwise, as we are switching to multistage-build.

Reply:
could you please elaborate what you refer to exactly when talking about linking in docker files?

`COPY --link` would still perform an initial copy and as we do not re-use copied contents (but build results), there is no benefit in doing so 🤔

Reply:
I actually meant _mount_ [ref](https://docs.docker.com/reference/dockerfile/#run---mount) for RUN-directives.